docs/flows.md

```diff
@@ -208,7 +208,7 @@ Flowdock-User: 2
       "email": "stevie@example.com",
       "avatar": "https://d2cxspbh1aoie1.cloudfront.net/5bdd089a099acc56fc7120f6325a5d5c/",
       "status": null,
-      "disabled": true,
+      "disabled": false,
```

Comment:
do we need to update this file:
https://github.com/flowdock/api-docs/blob/a026b6191883628f8d2943d72ca98f74678152fb/docs/users.md

Reply:
Nah, it doesn't mention anything about disabled users coming back in queries